src/main/java/org/springframework/data/elasticsearch/core/ReactiveDocumentOperations.java

```diff
@@ -86,6 +86,20 @@
 	 */
 	<T> Mono<T> save(T entity, IndexCoordinates index);
 
+	/**
+	 * Index entities under the given {@literal type} in the index extracted from entity metadata.
+	 *
+	 * @param entities must not be {@literal null}.
+	 * @param clazz the entity class
```

Comment:
the javadoc should reflect that the class parameter is used to determine the actual index.

Reply:
done